index.js

```diff
@@ -27,17 +38,32 @@ class RNCallKeep {
   addEventListener = (type, handler) => {
     const listener = listeners[type](handler);
 
-    this._callkeepEventHandlers.set(type, listener);
+    const listenerSet = this._callkeepEventHandlers.get(type) ?? new Set();
+    listenerSet.add(listener);
+
+    this._callkeepEventHandlers.set(type, listenerSet);
+
+    return new EventListener(type, listener);
```

Comment:
Can we add `this`  as a third argument here to avoid using the instance variable `callKeep` line 29 ?
Like:
```js
class EventListener {
  constructor(type, listener, callkeep) {
    this._type = type;
    this._listener = listener;
    this._callkeep = callkeep;
  }

  remove = () => {
    this._callkeep.removeEventListener(this._type, this._listener);
  };
}
// ...
return return new EventListener(type, listener, this);
```
And so we can remove the changes line 378 to 380.